tests/integration/new_relations/test_new_relations.py

```diff
@@ -668,6 +668,7 @@ async def test_discourse(ops_test: OpsTest):
 
 
 @pytest.mark.group(1)
+@pytest.mark.unstable
```

Comment:
Indico test fails due to the DB charm blocking over running out of space on the volume. Disabling to fix in a separate PR.